### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by imOlya**
- [DEMO LINK](https://imOlya.github.io/layout_catalog/)
- [TEST REPORT LINK](https://imOlya.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -21,7 +24,384 @@
...
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```
Comment (nazarmatsevych) [inline src/index.html:48]:
```suggestion
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
```

Add an empty line between multiple sibling blocks of html

File: src/index.html [hunk 2]
```diff
@@ -21,7 +24,384 @@
...
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
```
Comment (nazarmatsevych) [inline src/index.html:58]:
```suggestion
              href="#smartphones"
```

Fix href value everywhere

File: src/styles/blocks/header.scss [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      color: $hover-link-color;
+    }
+
+    &.is-active::after {
+      content: '';
+      position: absolute;
+      display: block;
+      bottom: 1;
+      background-color: $hover-link-color;
+      width: 37px;
```
Comment (nazarmatsevych) [inline src/styles/blocks/header.scss:73]:
```suggestion
      width: 100%;
```

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Almost done, let's make some imrpovements:

1. Make sure to add transition style under general selector, not the one with :hover - this way transition will work smoothly both ways.

**Review by GUSILLUS — APPROVED** [review]
Gj!

**Pull request closed without merging**